src/Core/Program.cs

```diff
@@ -621,5 +645,88 @@ public static bool GetCommandLineFlagAsBool(string key, bool def)
             var mode => throw new InvalidDataException($"Command line flag '{key}' value of '{mode}' is not valid")
         };
     }
+    public static void PrintCommandLineHelp()
+    {
+        Console.WriteLine("""
+Usage: SwarmUI.dll [option]...
+  or   SwarmUI.dll --help
+
+  Options:
```

Comment:
This should be put in some shared location so it's not duplicated between docs and here.
(Or just link to the docs from here lol)